README.md

```diff
@@ -36,6 +36,94 @@ See the Makefile for instructions.
 For further installation simply copy the resulting binary and man page
 (in the 'doc' directory) to appropriate directories.
 
+## Ringserver Container Image
+
+If you prefer to work with ringserver as a container, that is fully supported.
+Recent releases have containers available publicy as:
+
+[public.ecr.aws/earthscope/ringserver](https://gallery.ecr.aws/earthscope/ringserver)
+
+To pull this image with Docker and tag it locally as "ringserver" for convenience:
+
+```
+docker pull public.ecr.aws/earthscope/ringserver:latest
+docker tag public.ecr.aws/earthscope/ringserver:latest ringserver:latest
+```
+
+To run the default configuration of ringserver with Docker which listens locally on ports 16000 and 18000:
+
+```
+docker run -p 16000:16000 -p 18000:18000 ringserver
+```
+
+If you want the ring contents to persist container recreations, you will need to attach a [volume](https://docs.docker.com/storage/volumes/) to store the ring files. This can be done in several ways. Below illustrates using a subdirectory in the current working directory. Note that by default the container runs ringserver as user id 10000 so the directory on the local file system where the ring files are stored needs to be writable by that user id.
+
+```
+mkdir -p ring
+sudo chown -R 10000 ring
+docker run -p 16000:16000 -p 18000:18000 -v ${PWD}/ring:/data/ring ringserver
```

Comment:
It is possible to get an operation is not permitted error from this command if the ringserver directory under the Desktop directory on MacOSX Sonoma 14.4.1
```
 ringserver % sudo chown -R 10000 ring
 ringserver % sudo docker run -p 16000:16000 -p 18000:18000 -v ${PWD}/ring:/data/ring ringserver
   docker: Error response from daemon: error while creating mount source path '/host_mnt/PATH/ring': mkdir /host_mnt/USER/PATH/USER: operation not permitted.
```
If the Ringserver directory is moved to be lateral or above the Desktop directory, then this command returns a different set of errors:

```
sudo chown -R 10000 ring
ringserver % sudo docker run -p 16000:16000 -p 18000:18000 -v ${PWD}/ring:/data/ring ringserver
Tue Jul 30 15:54:52 2024 - ringserver version: 2020.075
Tue Jul 30 15:54:52 2024 - RingInitialize(): error opening ring/packetbuf: Permission denied
Tue Jul 30 15:54:52 2024 - Auto recovery, moving packet buffer and stream index files
Tue Jul 30 15:54:52 2024 - RingInitialize(): error opening ring/packetbuf: Permission denied
Tue Jul 30 15:54:52 2024 - Error re-initializing ring buffer on auto-recovery (-1)
```